submissions/chabuChabu/database/trips.sql

```diff
@@ -0,0 +1,37 @@
+/*The following query is used to create a table for storing passanger details*/
+use trips;
+create table passangers(
+id int not null auto_increment,
+fullname varchar(100) not null,
+sex varchar(10) not null,
+age int(11),
+primary key (id)
+);
+
+/*The following query is used to create a table for storing trip details*/
+/*It references the passanger table  via passanger ID*/
+create table trip_details(
+id int not null auto_increment,
+passanger_class int(11) not null,
+ticket_number varchar (50) not null,
+trip_fare decimal(10,2)not null,
+cabin varchar (50),
+parents int (11),
+children int (11),
+spouses int (11),
+siblings int (11),
+embarkation_point varchar(100) not null,
+passanger_id int not null,
+primary key(id),
+foreign key(passanger_id) references passangers(id)
+);
+
+/*The following query is used to create a table for storing information about accidents*/
+/*It references the passanger table via passanger ID*/
+create table accidents(
+id int not null auto_increment,
+status enum ('survived', 'died') not null,
```

Comment:
use `tinyint` instead

Reply:
@adaugochi which line are you referring too??

Reply:
line 33

Reply:
Already know that it is the line that I comment under that I am referring to. That's how comments are dropped on PR @ChabuChabu

Reply:
@adaugochi All right that's noted